### DISCUSSION TITLE
Registration broken?

### DISCUSSION THREAD

**Discussion by nakamigo** [vercel/community#29] [Help] [answered]
Tried to register using GitHub for the first time and failed: `User exists`
Then wrote an email to registration@vercel.com and got this:
```
From: Mail Delivery Subsystem <mailer-daemon@googlemail.com>

Hello,

We're writing to let you know that the group you tried to contact (registrations) may not exist, or you may not have permission to post messages to the group. A few more details on why you weren't able to post:

 * You might have spelled or formatted the group name incorrectly.
 * The owner of the group may have removed this group.
 * You may need to join the group before receiving permission to post.
 * This group may not be open to posting.

If you have questions related to this or any other Google Group, visit the Help Center at https://support.google.com/a/vercel.com/bin/topic.py?topic=25838.

Thanks,

vercel.com admins
```
Any ideas on what's happening?

**Comment by juliejonak** [comment, marked as answer]
Thanks for reporting! Can you test emailing to `registration@vercel.com` one more time? A test on our end went through successfully just now so that will help verify if it's still occurring for you.

Update: our team has solved this error and all emails will go through successfully

**Reply by nakamigo** [reply]
Got the same error just now `Delivery Status Notification (Failure)`, I am sending from Gmail
Same group https://support.google.com/a/vercel.com/bin/topic.py?topic=25838
How can I help you to narrow it down?

**Reply by Pieparker** [reply]
Are you sending to `registration@` or `registrations@`? Your original error looks like you tried `registrations` and I believe the address is `registration`. If you can try this, please do & report back!

**Reply by nakamigo** [reply]
Here is a snippet that's been sent back by Google after delivery failed
![image](https://user-images.githubusercontent.com/66538710/148284665-4e213c03-c186-4c76-9007-e69540d73850.png)

**Reply by juliejonak** [reply]
While we need to do some further investigating around what may have occurred here with the email, you should be able to register & log in now on Vercel. The original block may have occurred due to VPN usage or other IP flags, out of an abundance of caution. Please let me know if you run into any issues!

**Reply by juliejonak** [reply]
@metabuild8 You should also be good to go now too. Any chance you could verify for me the to: email you sent to and the error you received? We're investigating further and that will help us track down any patterns in both of these reports.

**Reply by juliejonak** [reply]
@metabuild8 Actually, all good. I found a way to replicate this on our end. Hopefully we'll have this solved shortly. Thank you for helping to report!
